man/man1/rtl_433.1

```diff
@@ -151,7 +151,7 @@ RTL\-SDR device driver is available.
 To set gain for RTL\-SDR use \-g <gain> to set an overall gain in dB.
 .RE
 .RS
-SoapySDR device driver is available.
+SoapySDR device driver is not available.
```

Comment:
Same here.

Reply:
It seems to be that the *.1 is a local generated file? 
And its coming from #ifdef SOAPYSDR
            "\tSoapySDR device driver is available.\n"
#else
            "\tSoapySDR device driver is not available.\n"
#endif

I can correct the Readme.md or what should I do?